src/openrc-shutdown/broadcast.c

```diff
@@ -144,10 +144,11 @@ void broadcast(char *text)
 
 	setutxent();
 
+	size_t len_path_dev = strlen(_PATH_DEV);
 	while ((utmp = getutxent()) != NULL) {
 		if (utmp->ut_type != USER_PROCESS || utmp->ut_user[0] == 0)
 			continue;
-		if (strncmp(utmp->ut_line, _PATH_DEV, strlen(_PATH_DEV)) == 0)
```

Comment:
`_PATH_DEV` is a constant string.  if the compiler can't optimize that to a constant integer at compile time, it's not worth caring about.  drop this change please.